src/sisl/io/wannier90/seedname.py

```diff
@@ -211,21 +199,13 @@ def read_geometry(self, *args, **kwargs):
         """Reads a `Geometry` and creates the Wannier90 cell"""
```

Comment:
add doc-string related to the order argument, also clarify that it will try and read from `_centres.xyz`

Reply:
changed the doc string here and updated most of them a bit